jenkins-jobs/nci/project.rb

```diff
@@ -54,6 +54,7 @@ def self.job(project, distribution:, architectures:, type:)
                                       architecture: architecture)
       scm = project.upstream_scm
       job.qt_git_build = (scm&.url&.include?('/qt/') && scm&.branch&.include?('5.15'))
+      job.qt6_build = (scm&.url&.include?('/qt-6/')
```

Comment:
```
job.qt6_build = (scm&.url&.include?('/qt-6/')
                ^ this is both unnecessary and never gets closed
```

Also, the SCM path doesn't include a dash, does it?

Reply:
no quite right, no hyphen.  and then took me far too long to realise brackets were unnecessary. 🤦